src/main/java/co/getmehired/getmehired/model/Talent.java

```diff
@@ -7,35 +7,265 @@
+		this.academicDegree = academicDegree;
+	}
+
+	public Citizenship getCitizenship() {
+		return citizenship;
+	}
+
+	public void setCitizenship(Citizenship citizenship) {
+		this.citizenship = citizenship;
+	}
+
+	public ImmigrationStatus getImmigrationStatus() {
+		return immigrationStatus;
+	}
+
+	public void setImmigrationStatus(ImmigrationStatus immigrationStatus) {
+		this.immigrationStatus = immigrationStatus;
+	}
+
+	public DegreeStatus getAccademicDegree() {
+		return accademicDegree;
+	}
+
+	public void setAccademicDegree(DegreeStatus accademicDegree) {
+		this.accademicDegree = accademicDegree;
+	}
+
+	public SupportNeeded getSupportNeeded() {
+		return supportNeeded;
+	}
+
+	public void setSupportNeeded(SupportNeeded supportNeeded) {
+		this.supportNeeded = supportNeeded;
+	}
+
+	public employmentStatus getEmploymentStat() {
+		return employmentStat;
+	}
+
+	public void setEmploymentStat(employmentStatus employmentStat) {
+		this.employmentStat = employmentStat;
+	}
+
+	public String getDegreeSubject() {
+		return degreeSubject;
+	}
+
+	public void setDegreeSubject(String degreeSubject) {
+		this.degreeSubject = degreeSubject;
+	}
+
+	public Date getGraduationDate() {
+		return graduationDate;
+	}
+
+	public void setGraduationDate(Date graduationDate) {
+		this.graduationDate = graduationDate;
+	}
+
+
+	
+	public Integer getSalaryStart() {
+		return salaryStart;
+	}
+
+	public void setSalaryStart(Integer salaryStart) {
+		this.salaryStart = salaryStart;
+	}
+
+	public String getCurrentJob() {
+		return currentJob;
+	}
+
+	public void setCurrentJob(String currentJob) {
+		this.currentJob = currentJob;
+	}
+
+	public String getCurrentEmployer() {
+		return currentEmployer;
+	}
+
+	public void setCurrentEmployer(String currentEmployer) {
+		this.currentEmployer = currentEmployer;
+	}
+
+	public Date getJobStartDate() {
+		return jobStartDate;
+	}
+
+	public void setJobStartDate(Date jobStartdate) {
+		this.jobStartDate = jobStartdate;
+	}
+
+	public String getJobStartDatestr() {
+		return jobStartDatestr;
+	}
 
+	public void setJobStartdatestr(String jobStartdatestr) {
+		this.jobStartDatestr = jobStartdatestr;
+	}
+
+	public Integer getJobSalary() {
+		return jobSalary;
+	}
+
+	public void setJobSalary(Integer jobSalary) {
+		this.jobSalary = jobSalary;
+	}
+
+	public String getNewEmployer() {
+		return newEmployer;
+	}
+
+	public void setNewEmployer(String newEmployer) {
+		this.newEmployer = newEmployer;
+	}
+
+	public String getNewPosition() {
+		return newPosition;
+	}
+
+	public void setNewPosition(String newPosition) {
+		this.newPosition = newPosition;
+	}
+
+	public String getAddress() {
+		if (this.address !=null) {
+			return address;
+		}
+		return "Address not found";
+	}
+	
+	public void setAddress(String address) {
+		this.address = address;
+	}
+	
+	public String getCalendlyUrl() {
+		if(this.calendlyUrl!=null) {
+			return calendlyUrl;
+		}
+		return "calendlyUrl not found";
```

Comment:
getter and setter should not have any logic. Their task is to only get and set data. The logic check should be done in controller.

Reply:
I solved this